config/environments/production.rb

```diff
@@ -50,7 +50,7 @@
 
   # Include generic and useful information about system operation, but avoid logging too much
   # information to avoid inadvertent exposure of personally identifiable information (PII).
-  config.log_level = :info
+  config.log_level = ENV.fetch("RAILS_LOG_LEVEL", "info")
```

Comment:
This is how the new rails template does it: https://github.com/rails/rails/blob/bab4aa7cb25112846e04cea907361a1de0f126ef/railties/lib/rails/generators/rails/app/templates/config/environments/production.rb.tt#L73C1-L73C58